Bhagavathi_Chirra/week1/loops-exercises/js/exercise1.js

```diff
@@ -0,0 +1,31 @@
+
+console.log('# Exercises: For loops');
+for(var i=0; i<=20; i++){
+  // if(i%2 == 0)
+  // {
+  //   console.log(`${i} is even`);
+  // }
+  // else {
+  //   console.log(`${i} is odd`);
+  // }
+  console.log(`${i} is ${ i%2 === 0 ? 'even' : 'odd'}.`);
+}
+
+console.log('## Multiplication Tables');
+for(var x=0; x<=10; x++){
+  for(var j=0; j<=10; j++){
+    console.log(`%c${x} * ${j} = ${x*j}`,'color:hotpink ; font');
+  }
+}
+
+console.log('## The Grade Assigner');
+for(var y=60; y<=100; y++)
```

Comment:
// I haven't run your Grade Assigner function, but if you would like to expand on it, you can create the solution as follows -
for ( score = 60; score <= 100; score++) {
  if (score >= 90){
    console.log(`For ${score}, you got an A`);
    //return 'A';
  }
  else if (score >= 75){
    console.log(`For ${score}, you got a B`);
  }
  else if (score >= 60){
    console.log(`For ${score}, you got a C`);
  }
  else if (score >= 50){
    console.log(`For ${score}, you got a D`);
  }
  else{
    console.log(`For ${score}, you got an F`);
    //return F;
  }
}